src/patternfly/components/Wizard/wizard-toggle-list.hbs

```diff
@@ -1,6 +1,6 @@
-<ol class="pf-c-wizard__toggle-list{{#if wizard-toggle-list--modifier}} {{wizard-toggle-list--modifier}}{{/if}}"
+<span class="pf-c-wizard__toggle-list{{#if wizard-toggle-list--modifier}} {{wizard-toggle-list--modifier}}{{/if}}"
```

Comment:
This looks good, but looks like the children of this element (`.pf-c-wizard__toggle-list-item`) are still `<li>`s, which is invalid in a `<span>`.